hooks/src/lib/cardNumber/useCardNumbers.ts

```
@@ -0,0 +1,81 @@
+    const { name, value } = event.target;
+    const errorType = validateCardNumbers(value);
+
+    if (restrictChange && errorType) {
+      return;
+    }
+
+    if (!restrictChange) {
+      setValidationResults((prev) => ({
+        ...prev,
+        [name]: {
+          isValid: !Boolean(errorType),
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)